### ISSUE TITLE
Continue training --epoch_count?

### ISSUE THREAD

**Issue by BanjerHD** [junyanz/pytorch-CycleGAN-and-pix2pix#906]
I am using Google Colab so I can only use it for 12 hours and then I have to wait to use it again for 12 hours, if I am right the progress is saved every 5 epochs? (around 1 hour in Google Colab, can I make this more often?)

I looked in the FAQ and it states:
> You can use the option --continue_train. Also set --epoch_count to specify a different starting epoch count. See more discussion in [training/test tips](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/docs/tips.md#trainingtest-tips).

So I looked at [training/test tips](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/docs/tips.md#trainingtest-tips) and it states:
> To fine-tune a pre-trained model, or resume the previous training, use the --continue_train flag. The program will then load the model based on epoch. By default, the program will initialize the epoch count as 1. Set --epoch_count <int> to specify a different starting epoch count.

What do I need to set <int> to continue training? Manually look how much epoch's it did and insert that? Or is it okay to use the default 1?

Thanks so much for helping!

**Comment by javergara** [comment]
by default, 200 epochs are selected, first 100 with the same learning rate and then 100 more to get it to 0.  you can see the parameters in https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/tree/master/options 

I just set --epoch_count 1  and done  but i did a backup just in case.

**Comment by BanjerHD** [comment]
> by default, 200 epochs are selected, first 100 with the same learning rate and then 100 more to get it to 0. you can see the parameters in https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/tree/master/options
> 
> I just set --epoch_count 1 and done but i did a backup just in case.

@javergara Thanks for your reply! Would setting '--epoch_count 1' overwrite the existing training done?

**Comment by junyanz** [comment]
You can use `--save_epoch_freq` to specify how frequently you save models (see this [line](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/options/train_options.py#L24)). 

When you continue the previous training, the program will load the model specified by `--epoch` (see this [line](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/options/base_options.py#L53)). 

Regarding `--epoch_count`, it decides the starting epoch count. See this [line](https://github.com/junyanz/pytorch-CycleGAN-and-pix2pix/blob/master/options/train_options.py#L27). If you stopped at 50th epoch in your previous experiments, you can set `--epoch_count 50`.
